client/src/resources/cert-and-project-map.ts

```diff
@@ -724,14 +712,8 @@ const certMap = [
   }
 ] as const;
 
-function getResponsiveWebDesignPath(
-  project: string,
-  { showNewCurriculum }: { showNewCurriculum: boolean }
-) {
-  // TODO: for the hard launch, the conditional should just be `showNewCurriculum`
-  return showNewCurriculum && showUpcomingChanges
-    ? `${responsiveWeb22Base}/${project}-project/${project}`
-    : `${responsiveWebBase}/${project}/`;
+function getResponsiveWebDesignPath(project: string) {
+  return `${responsiveWeb22Base}/${project}-project/${project}`;
```

Comment:
Sorry, I was typing on my phone and so couldn't easily select this bit to make a comment!

The issue is that `showUpcomingChanges` is always false in production, so this always returns `${responsiveWebBase}/${project}/`.  That's what we want until the hard launch, because it determines the links in `/settings` (and maybe other places, I forget)

Reply:
Ah, I see now - I'll fix this.